app/assets/javascripts/models/beta/projectBoard.js

```diff
@@ -36,6 +37,12 @@ export const isOperandSearch = word => {
 
 const containsInArray = (word, array) => array.some(operand => word.includes(operand))
 
+export const canCloseColumn = projectBoard =>
+  values(projectBoard.visibleColumns).filter(Boolean).length > 1;
+
+export const isClosing = (projectBoard, column) =>
```

Comment:
Maybe we could rename it to `isOpen` to make it make straight forward to get what that method returns?